assets/css/queries.css

```diff
@@ -0,0 +1,500 @@
+
+@media only screen and (min-width: 30em) {
+  .card {
+    border-radius: 0 0 3rem 3rem;
+    min-height: fit-content;
+  }
+}
+
+@media only screen and (min-width: 43em) {
+}
+
+@media only screen and (min-width: 58em) {
+}
+
+@media only screen and (min-width: 70em) {
```

Comment:
Si bien estos queries funcionan sin problema, nunca había visto que usaran `em` como unidad para definir el ancho de pantalla, ¿hay algún motivo por el cuál lo use así?

Reply:
Pues profe para serte sincero, es maña adquirida en el trabajo es básicamente para estandarizar los proyectos en la empresa, así evitamos trabajar con medidas estándar si no más adaptadas al requerimiento del cliente como un sello personal y para también optimizar el texto principalmente, ya que últimamente solo trabajo en páginas de noticias y blogs como el diario The guardian